tests/test.js

```diff
@@ -43,6 +43,44 @@ describe("Embroider build", function () {
   }
 });
 
+describe("When rootURL and publicPath do not match", function () {
```

Comment:
Would be great if you could assertition on `src`/`href` attribute value to double check the assumption that some of them are but some others aren't prefixed with `publicPath`. This will also help us catching when it's fixed upstream and this functionality may not be needed anymore.